### PR TITLE
Change document input placeholder

### LINKED ISSUE #119: Taxvat label change won't work on Firecheckout, since it uses placeholder instead of label

### PR DESCRIPTION
**Pull request by guilhermepiovesan**
closes #119

### CODE REVIEW

File: js/ebanx/document/document.js [hunk 1]
```diff
@@ -25,11 +26,15 @@ var changeTaxVatLabel = function () {
   var newLabel = getLabelByCountry(country, defaultLabel);
 
   taxVatLabel.innerHTML = newLabel;
+    if(taxVatInput) {
+      setTimeout(function(){taxVatInput.placeholder = newLabel;}, 10)
```
Comment (cezarlz) [inline js/ebanx/document/document.js]: Why do we need setTimeout?

File: js/ebanx/document/document.js [hunk 2]
```diff
@@ -25,11 +26,15 @@ var changeTaxVatLabel = function () {
...
+    }
 };
 
 var init = function () {
   var countrySelect = qs('#billing\\:country_id');
   taxVatLabel = qs('label[for="billing\\:taxvat"]');
+  taxVatInput = document.getElementById('billing:taxvat');
```
Comment (cezarlz) [inline js/ebanx/document/document.js]: Please, keep in mind that not always the `billing:taxvat` can be this value.

Comment (guilhermepiovesan) [inline js/ebanx/document/document.js]: Why not? billing:taxvat is default

File: js/ebanx/document/document.js (hunk 1 above)
Comment (guilhermepiovesan) [inline js/ebanx/document/document.js]: Magento has some code that generates the checkout fields again, so with setTimeout we solve this.

**Pull request merged**